### DISCUSSION TITLE
"Calibrate" Editor "Sliders" To Be More Accurate For Their Use Case

### DISCUSSION THREAD

**Discussion by Zenith222** [godotengine/godot-proposals#4928] [Editor]
![Screenshot 2022-07-20 at 12 33 42 PM](https://user-images.githubusercontent.com/26575338/180035642-b48eea08-c579-4e15-bf2e-811e4698819c.png)

As seen in the above screenshot, the slider to adjust the radius and height of the mesh instance is not very useful. A tiny movement of the slider will quickly set the values to a non-useful scale, and this could result in low performance, low accuracy adjustments, and perhaps even crashing. This problem permeates through many different sliders in Godot and I feel as if they often are not useful for this reason. The basic solution is to type in the desired value. However, the problem remains that a beginner could quickly crash the entire engine by increasing a value too quickly to an incredibly high level. Sliders should be adjusted to appropriate scales for each use case.

**Comment by Zireael07** [comment]
Can you list the sliders that have this problem? Also did you check on 4.x or 3.x? IIRC progress has been made on this...

**Comment by miv391** [comment]
How about making these kind of precise sliders? https://youtu.be/TnvCjziCUGI?t=167

Although Krita sliders still always jump to wherever the slider is clicked. I think it would be better not to jump the clicked position when shift is pressed. Just make small adjustements to the slider value to the direction where the mouse is dragged.

**Comment by timothyqiu** [comment]
See also #3524
